pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
+type Scale int
+
+const (
+	Nano  Scale = -9
+	Micro Scale = -6
+	Milli Scale = -3
```

Comment:
< 1 binary suffixes are *amazingly* confusing. No one will understand them or use them right. Please keep this list only 10^3n suffixes. (The binary suffixes are already too cute: .4GiB asks for a nibble and change, which no one would expect.)